compiler/rustc_abi/src/lib.rs

```diff
@@ -1401,6 +1401,7 @@ pub enum Variants<FieldIdx: Idx, VariantIdx: Idx> {
     /// Single enum variants, structs/tuples, unions, and all non-ADTs.
     Single { index: VariantIdx },
 
+    // REVIEW: probably this wording has to be adjusted somehow?
```

Comment:
Opening a conversation here so I don’t forget to resolve this somehow before merging.

Reply:
I'm going back and forth between just removing `inhabited` from the first sentence and documenting all the subtleties explicitly...

Reply:
let's just remove `inhabited` and land this PR